modules/persistence/src/services/pages/index.ts

```diff
@@ -144,6 +150,7 @@ class UpdatedPagesManager {
                 static_assets: this.findUpdatedAssets(page.static_assets, prevPageData?.static_assets),
                 updated_at: this.updateTime,
                 deleted: false,
+                ttl,
```

Comment:
this is adding a `ttl` field to the document.

i think the ticket calls for adding a `ttl index` to the whole collection. we can do this as a adhoc, or come up with a a manager of db and collections. i think the former would be faster in getting the needful done, but the latter would be a centralized place for db management